DependencyInjection/Configuration.php

```diff
@@ -52,6 +52,7 @@ public function getConfigTreeBuilder()
                 ->children()
                     ->scalarNode('from')->defaultValue('%sulu_admin.email%')->end()
                     ->scalarNode('to')->defaultValue('%sulu_admin.email%')->end()
+                    ->scalarNode('sender')->defaultValue('%sulu_admin.email%')->end()
```

Comment:
Sorry didn't see this before. I would set the defaultValue to `null` so its only set when configured by someone.